homeassistant/components/blebox/light.py

```diff
@@ -47,7 +47,7 @@ async def async_setup_entry(
     BleboxColorMode.RGB: ColorMode.RGB,
     BleboxColorMode.MONO: ColorMode.BRIGHTNESS,
     BleboxColorMode.RGBorW: ColorMode.RGBW,
-    # RGB and Brightness 2 and 3 implementation difference, if W hex is not null only this or RGB + Brightness separated with mask
+    # RGB and White, where white hex is prioritised over RGB channel
```

Comment:
Does the comment apply to the line above or to the line below?

Reply:
Normally a comment is put on the line above which it applies or on the same line.

If we can shorten it we may fit it on the same line.
Maybe just:
```py
# white hex is prioritised over RGB channel
```

Reply:
Got it, I'll use your suggestion.